rules_gapic/cpp/cc_gapic.bzl

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@com_google_api_codegen//rules_gapic:gapic.bzl", "gapic_srcjar", "unzipped_srcjar")
+
+def _cc_gapic_postprocessed_srcjar_impl(ctx):
+    gapic_zip = ctx.file.gapic_zip
+    output_main = ctx.outputs.main
+    output_main_h = ctx.outputs.main_h
+
+    output_dir_name = ctx.label.name
+    output_dir_path = "%s/%s" % (output_main.dirname, output_dir_name)
+
+    script = """
+    unzip -q {gapic_zip} -d {output_dir_path}
+    # TODO: Call formatter here
+    pushd .
```

Comment:
Nit: why not 
```
pushd {output_dir_path}
...
popd
```
?

Reply:
No particular reason, just an artifact from what used to be a bigger script. changed to `pushd {output_dir_path}`